### DISCUSSION TITLE
tooltips with overflow-hidden elements

### DISCUSSION THREAD

**Discussion by surrealroad** [huntabyte/shadcn-svelte#1570] [Help] [answered]
Hello
I don't think this is necessarily an issue, but I've noticed if I have an element that has `overflow-hidden` as a class, any tooltips within get clipped as well. Is there a way around this or do I just need to remove the overflow-hidden class?

**Comment by huntabyte** [comment, marked as answer]
Ensure you wrap the `Tooltip.Content` in `Tooltip.Portal` and that should alleviate that issue!

**Reply by surrealroad** [reply]
@huntabyte just a heads up that it looks like portal is not exported by default in /ui/tooltip/index.ts

**Comment by surrealroad** [comment]
thank you!
